contrib/endpoints/src/api_manager/api_manager_impl.cc

```diff
@@ -80,8 +86,7 @@ ApiManagerImpl::ApiManagerImpl(std::unique_ptr<ApiManagerEnvInterface> env,
     config_loading_status_ = utils::Status(Code::ABORTED, err_msg);
   }
 
-  check_workflow_ = std::unique_ptr<CheckWorkflow>(new CheckWorkflow);
-  check_workflow_->RegisterAll();
+  return config_loading_status_;
```

Comment:
We don't need a class variable to store this status.   The main reason to make it  in the Load() function so it can return the status right away,  not need to store it.